recipe/meta.yaml

```diff
@@ -1,53 +1,81 @@
-{% set version = "0.23.8" %}
+{% set name = "awscrt" %}
+{% set version = "0.27.4" %}
 
 package:
-  name: awscrt
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
-  sha256: cba55f3ee80ea3192a0a24e84caad778570250800a59d29ef9efbcd4d1612f2f
+  url: https://pypi.org/packages/source/a/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb23cb767ca2e3c0007c899b289d668d28aeb055534df35a741847135f6cd91c
   patches:
     - patches/0001-Don-t-force-static-linkage.patch
+    - patches/0002-disable-osx-failing-tests.patch  # [osx]
+    - patches/0003-disable-win-failing-test.patch  # [win]
 
 build:
   number: 0
+  skip: true  # [py<38]
+  script_env:
+    # Unvendor AWS libs
+    - AWS_CRT_BUILD_USE_SYSTEM_LIBS=1
+    - AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=1
+  # These are also in requirements: https://github.com/awslabs/aws-crt-python/blob/v0.27.4/setup.py#L220-L230
+  ignore_run_exports:
+    - aws-c-sdkutils
+    - aws-c-compression
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - patch     # [not win]
+    - patch  # [not win]
     - m2-patch  # [win]
   host:
     - python
     - pip
```

Comment:
Do we need `openssl` in `host` as you unvedor AWS libs? It provides `libcrypto.lib` https://conda-metadata-app.streamlit.app/Search_by_file_path?path=Library%2Flib%2Flibcrypto.lib